scripts/commands/updateApiDocs.ts

```diff
@@ -331,9 +331,10 @@ async function convertHtmlToMarkdown(
           : `/api/${pkg.name}/${link}`,
       );
 
-      path = currentReleaseNotesPath(pkg);
+      await writeReleaseNotes(pkg, result.markdown);
+    } else {
+      await writeFile(path, result.markdown);
     }
```

Comment:
Looking again at that `if` block, I think there's a bug. It wasn't doing what we wanted because, for runtime and provider, we are generating a release notes file even with the historical argument. I divided it into two early returns, one for the historical versions, and another for the separate release notes. The general case is writing the files as usual (including all API docs).